assets/scripts/popup.js

```diff
@@ -1,3 +1,5 @@
+var port = chrome.runtime.connect({name: "extensionOpen"});
```

Comment:
As the context here is interestingly complex, we can avoid some future problems by checking the possibility of changing the `port` variable declaration to `let` or `const`. By declaring it as `var` we run the 'risk' of making it public in a possible context different from the one we are working with. Do we really want this?